src/components/MenuItem.js

```diff
@@ -235,10 +251,15 @@ const MenuItem = React.forwardRef((props, ref) => {
                                             </Text>
                                         )}
                                         <View style={[styles.flexRow, styles.alignItemsCenter]}>
-                                            {Boolean(props.title) && Boolean(props.shouldRenderAsHTML) && <RenderHTML html={convertToLTR(props.title)} />}
-
-                                            {Boolean(props.shouldParseTitle) && Boolean(html.length) && !props.shouldRenderAsHTML && <RenderHTML html={`<comment>${html}</comment>`} />}
-
+                                            {Boolean(props.title) &&
+                                                (Boolean(props.shouldRenderAsHTML) || (Boolean(props.shouldParseTitle) && Boolean(html.length))) &&
+                                                (isNative ? (
+                                                    <RenderHTML html={getProcessedTitle} />
+                                                ) : (
+                                                    <View style={styles.chatItemMessage}>
+                                                        <RenderHTML html={getProcessedTitle} />
+                                                    </View>
+                                                ))}
```

Comment:
I missed this in my initial review, but writing platform specific code like this is against our guidelines